devicetypes/smartthings/zwave-rgbw-controller.src/zwave-rgbw-controller.groovy

```diff
@@ -0,0 +1,241 @@
+/**
+ *  Copyright 2018 SmartThings
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License. You may obtain a copy of the License at:
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software distributed under the License is distributed
+ *  on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License
+ *  for the specific language governing permissions and limitations under the License.
+ *
+ *  Z-wave RGBW Controller (White chanel is dimmed and doesn't support color temperature)
+ *
+ *  Author: SmartThings / vseven (Allan)
+ *  Date: 2018-10-26
+ */
+
+metadata {
+	definition (name: "ZWave RGBW Controller", namespace: "smartthings", author: "SmartThings", ocfDeviceType: "oic.d.light", mnmn: "SmartThings", vid: "generic-rgbw-color-bulb") {
```

Comment:
if it doesn't support color temperature it should just be "RGB"

Reply:
But it does support color temperature? Maybe it means something different to you, but we use "color temperature" and "white color" interchangeably.

Reply:
Color temperature to me means actual color temperature, 2700 K through 6500 K usually. However most rgbw controllers simply have a white channel that is 0 to 100% and acts like the R G & B channels.

Reply:
Ok - Got a better answer for you after some testing.  On a RGBW "controller", the one I'm working on and a Fibaro and I imagine any other controller (non-bulb).  For bulbs color temperature is sent as a cool white and warm white value which are the inverse of each other.  But on the controller it just has a white channel.  So at 2700k with this DTH its sending the cool white channel 100% and the warm 0%.  The problem is this puts a RGBW controller to 100% white.  As you slide the slider "warmer" to 6500K the white channel drops to 0%.  So what you see on the screen is the opposite of what is actually happening.

For that reason you do need a RGBW Controller DTH in addition to a RGBW bulb DTH.  But like I said I'll let you review the changes to RGBW-Light and once its good copy/paste into a controller DTH and adjust that color temp to operate the white channel correctly.  I'll also remove he color temp capability since it doesn't have it.

With that said there are RGBWW controllers out there and analog RGBWW led strips that do have "cool" and "warm" white leds which would work correctly on the RGBW-Light DTH.  But I've never seen anyone using them and if someone was their fingerprint could just be added to RGBW-Light.